shell/platform/fuchsia/dart_runner/main.cc

```diff
@@ -34,20 +31,17 @@ static void RegisterProfilerSymbols(const char* symbols_path,
 #endif  // !defined(DART_PRODUCT)
 
 int main(int argc, const char** argv) {
+  fx_log_init();
   async::Loop loop(&kAsyncLoopConfigAttachToCurrentThread);
 
-#if !defined(FUCHSIA_SDK)
-  syslog::InitLogger();
-
   std::unique_ptr<trace::TraceProviderWithFdio> provider;
   {
-    TRACE_EVENT0("dart", "CreateTraceProvider");
+    TRACE_DURATION("dart", "CreateTraceProvider");
```

Comment:
Is this the only place this update was needed? I see a TRACE_EVENT1 in dart_runner.cc. Is that okay?

Reply:
Yup, `TRACE_EVENT1` is fine.